src/Utils/ForeignKeyData.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin\SqlParser\Utils;
+
+final class ForeignKeyData
```

Comment:
```suggestion
final class ForeignKey
```

I prefer only ForeignKey, as it's way easier to change a class name in PMA than in this lib.